12주차/(P)17686/(P)17686_노유빈_python.py

```diff
@@ -0,0 +1,91 @@
+# 실패 : 몇 가지의 경우에서 답이 다르게 출력이 된다. "비교해주며 순서를 바꿔야 할 상황이 오면 a,b,c 모두 자리를 바꿔주기" 부분에서 문제가 있는데 어느 부분인지 못찾겠다.
+
+
+
+def find_HEAD(n): # HEAD를 return 하는 함수
+    head = ""
+    for i in range(0,len(list(n))):
+        for j in range(0,len(list(n[i]))):
+            if n[i][j].isdigit()!=True:
```

Comment:
```suggestion
    for file in n:
        if not file[i].isdigit():
```
1. for문은 시작 범위를 주지 않으면 0부터 시작해요.
2. list로 변환하지 않아도 문자열 그대로 index 탐색 가능해요.
```python
x = 'abc'
for i in x:
    print(i)

'''
result:
a
b
c
'''
```
3. range를 사용하지 않고 이런 방식으로도 for문을 사용할 수 있어요!